delivery_package_number/README.rst

```diff
@@ -81,16 +81,18 @@ Contributors
 
 -  `Tecnativa <https://www.tecnativa.com>`__:
 
-<<<<<<< HEAD
-  * Pedro M. Baeza
-  * David Vidal
-  * Carlos Roca
-  * Sergio Teruel
+   -  Pedro M. Baeza
+   -  David Vidal
+   -  Marçal Isern
+   -  Carlos Roca
```

Comment:
You should have resolve merge conflicts in the previous commit and don't let them show in the diff...